### PR TITLE
App Service sample deployment and ARM template

### PR DESCRIPTION
**Pull request by balteravishay**
This PR adds initial docs on how to use Azure App Service with az cli or with ARM template to host presidio services.
* the readme should finally be in the sample folder.
* the ARM template will be used for deploy-to-azure buttons on both services

### CODE REVIEW

**Review by navalev — APPROVED** [review]
approving with minor comments on the readme

File: docs/deployment-samples/deploy-to-app-service.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+##TODO: NOT WORKING CHECK WHY THE #APP_SERVICE_ID not works!!!
+az monitor diagnostic-settings create --name $APP_SERVICE_NAME-diagnostics --resource /
+$APP_SERVICE_ID --logs   '[{"category": "AppServicePlatformLogs","enabled": true}, {"category": "AppServiceConsoleLogs", "enabled": true}]' --metrics '[{"category": "AllMetrics","enabled": true}]' --workspace $LOG_ANALYTICS_WORKSPACE_ID
+```
+
+## Using an ARM template
+
+Alternatlively, you can use the provided ARM template which uses either an existing App Service Plan, or creates a new one.
+Note that while Log Analytics integration with Azure App Service is in preview, the ARM template deployment will not create a Log Analytics resource or configure the diagnostics settings from the App Service to a Log Analytics workspace.
+
```
[the remark this answers is not in the document]
Comment (balteravishay) [reply docs/deployment-samples/deploy-to-app-service.md:74]: can we review this request in the next PR which adds the deploy to azure buttons?

[the remark this answers is not in the document]
Comment (navalev) [reply docs/deployment-samples/deploy-to-app-service.md:76]: sure

**Review by navalev — APPROVED** [review]

File: docs/deployment-samples/deploy-to-app-service.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# create a log analytics workspace
+az monitor log-analytics workspace create --resource-group $LOG_ANALYTICS_WORKSPACE_RESROUCE_GROUP --workspace-name $LOG_ANALYTICS_WORKSPACE_NAME
+
+# query the log analytics workspace id
+LOG_ANALYTICS_WORKSPACE_ID=$(az monitor log-analytics workspace show --resource-group $LOG_ANALYTICS_WORKSPACE_RESROUCE_GROUP --workspace-name $LOG_ANALYTICS_WORKSPACE_NAME --query id)
+# query the app service id
+APP_SERVICE_ID=$(az monitor log-analytics workspace show --resource-group $RESOURCE_GROUP --name $APP_SERVICE_NAME --query id)
+
+# create the diagnostics settings
+##TODO: NOT WORKING CHECK WHY THE #APP_SERVICE_ID not works!!!
```
Comment (shiranr) [inline docs/deployment-samples/deploy-to-app-service.md:66]: Is this still needed?

**Review by shiranr — DISMISSED** [review]
Looks good. small comment about the remark.

**Review by shiranr — APPROVED** [review]

**Review by navalev — APPROVED** [review]

**Pull request merged**